demisto_sdk/tests/test_files/content_slim/Packs/Sample01/README.md

```diff
@@ -59,4 +59,94 @@ If you are interested in contributing, visit our contribution process [here](htt
 
 Thank you for contributing to Cortex XSOAR.
 
+If you are interested in contributing, visit our contribution process [here](https://xsoar.pan.dev/docs/contributing/contributing).#### This pack was co-authored by:
```

Comment:
why did you modify this file?

Reply:
Not sure how this was modified, reverting

Reply:
Seems this was not reverted